packages/@aws-cdk/aws-apigatewayv2/lib/http/authorizer.ts

```diff
@@ -78,7 +134,7 @@ export interface HttpAuthorizerAttributes {
   /**
    * Type of authorizer
    */
-  readonly authorizerType: HttpAuthorizerType
+  readonly authorizerType: HttpRouteAuthorizerType
```

Comment:
Why change this name?

Reply:
Not a name change. The values for the enums are different.